ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__ocp-installer-rehearse.yaml

```diff
@@ -39,11 +39,11 @@ tests:
   steps:
     cluster_profile: azure4
     workflow: cucushift-installer-rehearse-azure-ipi-sharednetwork
-- as: installer-rehearse-gcp
+- as: installer-rehearse-gcp-ipi-private
```

Comment:
`installer-rehearse-gcp` was placed there more like a placeholder, and we also mentioned that as a guide in https://issues.redhat.com/browse/OCPQE-7091?focusedCommentId=19367918&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-19367918. So I suggest not changing the job's name here.

Reply:
Do you mean, to add another section for the new test?

Reply:
I meant leave this line untouched.